scripts/fix/diplomats.lua

```diff
@@ -0,0 +1,103 @@
+-- Add Elven diplomats to negotiate tree caps
+--[[=begin
+
+fix/diplomats
+=============
+Adds a Diplomat position to all Elven civilizations, allowing them to negotiate
+tree cutting quotas - and you to violate them and start wars.
+This was vanilla behaviour until ``0.31.12``, in which the "bug" was "fixed".
+
+=end]]
+
+
+function update_pos(pos, ent) do
```

Comment:
You don't need the "do" here. (If you want it, you'll need an extra "end" at the end of this function.)

Incidentally, this is why the travis build is failing - this is a syntax error, so this script can't possibly work. Maybe you made a last-minute change or something, but we'd really generally prefer that you make sure things work before you make a PR.